pyDeltaRCM/water_tools.py

```diff
@@ -42,7 +27,7 @@ def init_water_iteration(self):
 
     def run_water_iteration(self):
 
-        iter = 0
+        _iter = 0
```

Comment:
Oh yeah. This definitely needed to be changed, since `iter` is actually a reserved word right?